### PR TITLE
Samsung.OrsaySDK version 5.1

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98732)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @upintheairsheep,

The installation isn't unattended:
image

Is there an installer switch to bypass this and have it install automatically?

**Comment by upintheairsheep** [comment]
> Hi @upintheairsheep,
> 
> The installation isn't unattended: image
> 
> Is there an installer switch to bypass this and have it install automatically?

Im not sure

**Comment by upintheairsheep** [comment]
This feels like a VM issue

**Comment by stephengillie** [comment]
> This feels like a VM issue

Sorry, it's a dependency on a JRE such as `Oracle.JDK.19`.

**Comment by upintheairsheep** [comment]
Add the dependencies tag

**Comment by ItzLevvie** [comment]
[Policy] Dependencies

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

File: manifests/s/Samsung/OrsaySDK/5.1/Samsung.OrsaySDK.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.OrsaySDK
+PackageVersion: 5.1
```
Comment (stephengillie) [inline manifests/s/Samsung/OrsaySDK/5.1/Samsung.OrsaySDK.installer.yaml:5]:
```suggestion
PackageVersion: 5.1
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Oracle.JavaRuntimeEnvironment
```

**Comment by stephengillie** [comment]
With the dependency, the install isn't unattended: 
image

**Pull request closed without merging**